ssvlib: update grid

thanhtung2705 opened the pull request.

thanhtung2705 committed:
ssvlib: update grid

bienvu commented on ssvlibraries/pattern-lab/source/_patterns/01-components/grid/grid--2cols.twig:
```diff
@@ -0,0 +1,21 @@
+<div class="grid grid--2cols">
+  <div class="container">
```
remove container

bienvu commented on ssvlibraries/pattern-lab/source/_patterns/01-components/grid/grid--4cols.twig:
```diff
@@ -0,0 +1,37 @@
+  <div class="container">
+    <div class="grid__wrap">
+      <div class="grid__item">
+        <img src="../../images/riseandshine-mattresses.jpg" alt="">
```
change name of image to grid-image

bienvu commented on ssvlibraries/scss/components/grid.scss:
```diff
@@ -0,0 +1,55 @@
+  }
+
+  &__item {
+    width: calc((100% - 61px) / 3);
```
please create a new variable $margin: 30px;
width: calc((100% - ($margin * 2 + 1) / 3);

bienvu commented on ssvlibraries/scss/components/grid.scss:
```diff
@@ -0,0 +1,55 @@
+    }
+  }
+
+  &--2cols {
```
Please comment before variable and fix responsive for this component

thanhtung2705 committed:
ssvlib: edit col components

lehung1109 committed:
ssvlib: update mixins grid

ssvlib: upload _test.scss

ssvlib: update mixin

ssvlib: update mixxin

ssvlib: update mixin
